web/src/EntryPage.js

```diff
@@ -94,6 +95,14 @@ class EntryPage extends React.Component {
         });
     };
 
+    if (this.state.application?.isRestricted) {
```

Comment:
Change to Application.BannedIp (string), so we can store the client IP address here to show it in error message